lib/voice/VoiceConnection.js

```diff
@@ -163,6 +174,10 @@ class VoiceConnection extends EventEmitter {
         return this.piper.volumeLevel;
     }
 
+    #aes256Available() {
+        return crypto && crypto.getCiphers().includes("aes-256-gcm");
```

Comment:
it's 2024, use optional chaining
```suggestion
        return crypto?.getCiphers().includes("aes-256-gcm");
```